plugins/a11yhelp/plugin.js

```diff
@@ -22,30 +22,148 @@
 
 		init: function( editor ) {
 			var plugin = this;
-			editor.addCommand( commandName, {
-				exec: function() {
-					var langCode = editor.langCode;
-					langCode =
-						plugin.availableLangs[ langCode ] ? langCode :
-						plugin.availableLangs[ langCode.replace( /-.*/, '' ) ] ? langCode.replace( /-.*/, '' ) :
-						'en';
-
-					CKEDITOR.scriptLoader.load( CKEDITOR.getUrl( plugin.path + 'dialogs/lang/' + langCode + '.js' ), function() {
-						editor.lang.a11yhelp = plugin.langEntries[ langCode ];
+			var langCode = editor.langCode;
+			langCode =
+				plugin.availableLangs[ langCode ] ? langCode :
+				plugin.availableLangs[ langCode.replace( /-.*/, '' ) ] ? langCode.replace( /-.*/, '' ) :
+				'en';
+			CKEDITOR.scriptLoader.load( CKEDITOR.getUrl( plugin.path + 'dialogs/lang/' + langCode + '.js' ), function() {
+				editor.lang.a11yhelp = plugin.langEntries[ langCode ];
+
+				editor.addCommand( commandName, {
+					exec: function() {
+						CKEDITOR.document.appendStyleSheet( CKEDITOR.getUrl( plugin.path + 'styles/a11yhelp.css' ) );
 						editor.openDialog( commandName );
-					} );
-				},
-				modes: { wysiwyg: 1, source: 1 },
-				readOnly: 1,
-				canUndo: false
-			} );
+					},
 
-			editor.setKeystroke( CKEDITOR.ALT + 48 /*0*/, 'a11yHelp' );
-			CKEDITOR.dialog.add( commandName, this.path + 'dialogs/a11yhelp.js' );
+					modes: { wysiwyg: 1, source: 1 },
+					readOnly: 1,
+					canUndo: false,
+					label: editor.lang.a11yhelp.commandLabel
+				} );
+				editor.setKeystroke( CKEDITOR.ALT + 48 /*0*/, 'a11yHelp' );
+				CKEDITOR.dialog.add( commandName, plugin.path + 'dialogs/a11yhelp.js' );
+			} );
 
 			editor.on( 'ariaEditorHelpLabel', function( evt ) {
 				evt.data.label = editor.lang.common.editorHelp;
 			} );
 		}
 	} );
+
+	/**
+	 * Set of a11yHelp plugin helpers.
+	 *
+	 * @class
+	 * @singleton
+	 * @since 4.8.0
+	 */
+	CKEDITOR.plugins.a11yhelp = {
+		/**
+		 * Return html which represent keystroke. Result is wrapped with `<kbd></kbd>` tags.
+		 *
+		 * 		CKEDITOR.plugins.a11yhelp.representKeystroke( CKEDITOR.instances.editor, 4456496 )
+		 * 		"<kbd><kbd>Alt</kbd>+<kbd>0</kbd></kbd>"
+		 *
+		 * @since 4.8.0
+		 * @param {CKEDITOR.editor} editor Editor instance.
+		 * @param {Number} keystroke Number which represent command keystroke.
+		 * @returns {String} HTML code with key names wrappeed with `<kbd></kbd>` tags.
+		 */
+		representKeystroke: function( editor, keystroke ) {
+			var lang = editor.lang.a11yhelp,
+				coreLang = editor.lang.common.keyboard;
+
+			// CharCode <-> KeyChar.
+			var keyMap = {
```

Comment:
Maybe this map could be defined as `CKEDITOR.plugins.a11yhelp` property? Now, every time the method is called, the map is defined again.